superset/db_engine_specs.py

```diff
@@ -650,12 +673,21 @@ def handle_cursor(cls, cursor, query, session):
         while polled:
             # Update the object and wait for the kill signal.
             stats = polled.get('stats', {})
+            processed_rows = stats['processedRows']
 
             query = session.query(type(query)).filter_by(id=query.id).one()
             if query.status == QueryStatus.STOPPED:
                 cursor.cancel()
                 break
 
+            if (
+                config.get('PREFETCH_PRESTO') and
+                processed_rows > 1000 and
```

Comment:
Why is this a hard coded value? Shouldn't this be tied to the `limit` defined in the `prefetch_results` method?